study.md

```diff
@@ -31,7 +31,9 @@ manipulates it by sorting it and then reversing it in place (i.e., modifying the
 reference to the original).
 
 ```ruby
-# your answer here
+example = [23, 56, 3, 7]
+example.sort.reverse
```

Comment:
Since we want to modify the original array, the calls to `sort` and `reverse` should be `sort!` and `reverse!`, respectively.